src/pages/ambassadors.tsx

```diff
@@ -37,6 +37,11 @@ function AmbassadorView(props) {
                                 </q>
                             </Typography> : <></>
                         }
+                        {person.clubs ?
+                            <Typography variant={"subtitle2"}>
+                                {person.clubs}
```

Comment:
what about this?
`
person.clubs.maps( (item) => {
<Chips label={item} />
})
`